docs/customizations/livechatwidget.md

```diff
@@ -0,0 +1,265 @@
+| loadingPaneProps | [ILoadingPaneProps](./loadingpane.md) | No | Controls the properties of the loading pane | [defaultLoadingPaneProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/loadingpane/common/defaultProps/defaultLoadingPaneProps.ts)
+| outOfOfficeChatButtonProps | [IChatButtonProps](./chatbutton.md) | No | Controls the properties of the chat button when the widget is outside operating hours | See [Chat Button](./chatbutton.md)
+| outOfOfficeHeaderProps | [IHeaderProps](./header.md) | No | Controls the properties of the header when the widget is outside operating hours | See [Header](./header.md)
+| outOfOfficeHoursPaneProps | [IOOOHPaneProps](./ooohpane.md) | No | Controls the properties of the out of operating hours pane, when chat button is clicked outside of operating hours | [defaultOOOHPaneProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/outofofficehourspane/common/defaultProps/defaultOOOHPaneProps.ts)
+| postChatLoadingPaneProps | [ILoadingPaneProps](./loadingpane.md) | No | Controls the properties of the loading pane before showing post chat survey | See [PostChatLoadingPane](./postchatloadingpane.md)
+| postChatSurveyPaneProps | [IPostChatSurveyPaneStatefulProps](./postchatsurveypane.md) | No | Controls the properties of the post chat survey pane, after the chat ends | See [PostChatSurvey](./postchatsurvey.md)
+| preChatSurveyPaneProps | [IPreChatSurveyPaneProps](./prechatsurveypane.md) | No | Controls the properties of the post chat survey pane, before the chat starts | [defaultPreChatSurveyPaneProps](../../chat-components/src/components/prechatsurveypane/common/defaultProps/defaultPreChatSurveyPaneProps.ts)
+| proactiveChatPaneProps | [IProactiveChatPaneStatefulProps](./proactivechat.md) | No | Controls the properties of the proactive chat pane | [defaultProactiveChatPaneProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/proactivechatpane/common/default/defaultProps/defaultProactiveChatPaneProps.ts)
+| reconnectChatPaneProps | [IReconnectChatPaneStatefulProps](./reconnectchatpane.md) | No | Controls the properties of the reconnect pane, when an auth chat user returns to the chat before the Reconnect timer expires | [defaultReconnectChatPaneProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/reconnectchatpane/common/default/defaultProps/defaultReconnectChatPaneProps.ts)
+| startChatErrorPaneProps | [IStartChatErrorPaneProps](./startchaterrorpane.md) | No | Controls the properties of the pane that shows up when start chat process has an error and is unble to proceed to conversation | -
```

Comment:
Broken link, seems we dont have the startChatErrorPane file.

Reply:
removed link